demos/python_demos/whiteboard_inpainting_demo/whiteboard_inpainting_demo.py

```diff
@@ -87,14 +91,18 @@ def main():
                         help='Optional. List of monitors to show initially')
     args = parser.parse_args()
 
-    capture = VideoCapture(args.i)
+    cap = open_images_capture(args.input, args.loop)
+    frame = cap.read()
+    if frame is None:
+        raise RuntimeError("Can't read an image from the input")
 
     if bool(args.m_instance_segmentation) == bool(args.m_semantic_segmentation):
         raise ValueError('Set up exactly one of segmentation models: '\
                          '--m_instance_segmentation or --m_semantic_segmentation')
 
-    frame_size, fps = capture.get_source_parameters()
-    out_frame_size = (int(frame_size[0]), int(frame_size[1] * 2))
+    frame_size = frame.shape
+    fps = cap.fps()
+    out_frame_size = (int(frame_size[1]), int(frame_size[0] * 2))
```

Comment:
```suggestion
    out_frame_size = (frame_size[1], frame_size[0] * 2)
```
shape contains ints.